tests/vmi_multus_test.go

```diff
@@ -864,29 +864,44 @@ var _ = Describe("[Serial]SRIOV", func() {
 			waitVmi(vmi1)
 			waitVmi(vmi2)
 
-			vmi1, err = virtClient.VirtualMachineInstance(vmi1.Namespace).Get(vmi1.Name, &metav1.GetOptions{})
-			Expect(err).NotTo(HaveOccurred())
-			vmi2, err = virtClient.VirtualMachineInstance(vmi2.Namespace).Get(vmi2.Name, &metav1.GetOptions{})
-			Expect(err).NotTo(HaveOccurred())
+			Eventually(func() string {
```

Comment:
Instead adding here Eventually, this is a larger problem
that affect more tests,
Since the guest agent takes time to report the interfaces, and the ready event is not enough to know the status has the info, it might flake.

I suggest alternative a robust to this one,
Add a function that will make sure that the expected fields are reported, as an helper function,
that function would be used on all the tests of that depend on the status.

Reply:
+1 for that instead of the eventually

Reply:
it will also use Eventually but in a dedicated general helper

Reply:
I agree this deserves a different PR, as it could help with many tests, and just related to the tests covered here.

Reply:
meanwhile i will start it, this PR might be a nice chance to introduce it, as for the network name we must get the MAC, and before validating we can get the expected network name according the MAC, we need to know the info exists.

Reply:
Done, added waitForVMIGuestAgentReport